components/BoardMember.vue

```diff
@@ -0,0 +1,101 @@
+<template>
+    <li class="py-4 md:py-0">
+        <div class="md:grid md:grid-cols-3 lg:gap-8 sm:pb-2">
+            <div>
+                <img
+                    class="w-full object-cover shadow-lg rounded-lg md:h-60"
+                    :src="`${imageLink}`"
+                    alt=""
+                />
+            </div>
+
+            <div class="md:col-span-2">
+                <div class="space-y-1">
+                    <div class="text-lg font-medium">
+                        <h3>{{ name }}</h3>
+                        <p class="text-indigo-600">{{ title }}</p>
+                        <div v-if="linkedInAccount" class="flex pb-1">
+                            <span class="sr-only">linkedIn</span>
+                            <a :href="linkedInAccount">
+                                <svg
+                                    class="inline w-5 h-5"
+                                    fill="#c0c0c0"
+                                    viewBox="0 0 24 24"
+                                    aria-hidden="true"
+                                    preserveAspectRatio="xMidYMid meet"
+                                >
+                                    <path
```

Comment:
I almost guarentee we'll never find some of these SVGs again when we're looking. Can you mention in the `README` where you got the SVGs from. Perhaps just add a `RESOURCES` section at the bottom?